### ISSUE TITLE
[Feature] Right / Left stick assignment with gyro control

### ISSUE THREAD

**Issue by Tak-attack** [redphx/better-xcloud#233]
**I want to suggest this feature:**  
If the stick can be controlled by the gyro function of the mobile device in the same way that the control stick is controlled by the mouse, it may make it easier to play FPS, TPS, racing games, etc.

Can you consider this?

**Comment by redphx** [comment]
Yes, I thought about this before. But having gyro control all the time will be annoying. Maybe only activate it when holding LT?

**Comment by Tak-attack** [comment]
If it is FPS/TPS, assigning to LT would be better.
If possible, I would like to be able to change it to button/trigger/always in the settings.
I would also like to see an invert setting. (e.g. turn off gyro only while holding)
